crates/pessimistic-proof/src/certificate.rs

```diff
@@ -1,71 +1,35 @@
-use std::collections::BTreeMap;
-
 use serde::{Deserialize, Serialize};
 
-use crate::{
-    bridge_exit::NetworkId,
-    keccak::Digest,
-    local_balance_tree::{BalanceTree, BalanceTreeByNetwork},
-    local_exit_tree::{hasher::Keccak256Hasher, LocalExitTree},
-    BridgeExit,
-};
+use crate::{bridge_exit::NetworkId, keccak::Digest, BridgeExit};
 
-/// Represents the required data from each CDK for the pessimistic proof.
+/// Represents the data submitted by the CDKs to the AggLayer.
 #[derive(Clone, Debug, Serialize, Deserialize)]
 pub struct Certificate {
     /// Origin network which emitted this certificate
     pub origin_network: NetworkId,
-    /// Initial local exit tree
-    pub prev_local_exit_tree: LocalExitTree<Keccak256Hasher>,
     /// Initial local exit root
     pub prev_local_exit_root: Digest,
-    /// Initial balance tree
-    pub prev_local_balance_tree: BalanceTree,
     /// Set of bridge exits
     pub bridge_exits: Vec<BridgeExit>,
+    /// Set of imported bridge exits
```

Comment:
Could we have more doc about those imported_bridge_exits?

Reply:
Added more docs